CommonTools/RecoUtils/src/precompile.h

```diff
@@ -0,0 +1,13 @@
+// for integration test 
```

Comment:
Can you get it to compile with `-e` so that we have the preprocessed 
file?

On 3 Feb 2015, at 12:22, David Lange wrote:

>> @@ -0,0 +1,13 @@
>> +// for integration test
>
> actually removing just the line for TrackExtra.h makes no difference - 
> are we pulling in all headers from a package in some way?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-sw/cmssw/pull/7480/files#r23997429